pubtools/pulplib/_impl/model/unit/file.py

```diff
@@ -31,6 +32,17 @@ class FileUnit(Unit):
         default="iso", type=str, pulp_field="_content_type_id"
     )
 
+    repository_memberships = pulp_attrib(
+        default=attr.Factory(FrozenList),
+        type=list,
+        converter=FrozenList,
+        pulp_field="repository_memberships",
+    )
+    """List of repositories which unit belongs to
```

Comment:
When I added this to unit base class attr lib complained about mandatory attributes were defined after optional one is defined. It was just a warning but I wanted to avoid that.